packages/core/src/recon.ts

```
@@ -47,10 +49,39 @@ export class ReconApi implements IReconApi {
   }
 
   async init(): Promise<void> {
+    if (!this.enabled) {
+      return
+    }
+
     this.#url = await this.#config.url
+    await this.registerInterest(Model.MODEL)
+  }
+
+  async registerInterest(model: StreamID): Promise<void> {
+    if (!this.enabled) {
+      this.#logger.imp(`Recon: disabled, not registering interest in model ${model.toString()}`)
```

maybe throw an error?